docs/core/install/runtime.md

```diff
@@ -37,7 +37,32 @@ macOS has standalone installers that can be used to install the .NET Core 3.1 ru
 
 ## Install with a package manager
 
-You can install the .NET Core Runtime with many of the common Linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-manager-rhel7.md).
+You can install the .NET Core Runtime with many of the common Linux package managers. For more information, see [Linux Package Manager - Install .NET Core](linux-package-managers.md).
+
+Installing with a package manager is only supported on the x64 architecture. If you're installing the .NET Core Runtime with a different architecture, such as ARM, follow the [Download and manually install](#download-and-manually-install) instructions below. For more information about what architectures are supported, see [.NET Core dependencies and requirements](dependencies.md).
+
+## Download and manually install
+
+To extract the SDK and make the commands available at the terminal, first [download](#all-net-core-downloads) a .NET Core binary release. Then, open a terminal and run the following commands.
+
+```bash
+mkdir -p $HOME/dotnet && tar zxf aspnetcore-runtime-3.1.0-linux-x64.tar.gz -C $HOME/dotnet
+export DOTNET_ROOT=$HOME/dotnet
+export PATH=$PATH:$HOME/dotnet
+```
+
+> [!TIP]
+> The above `export` commands will only make the .NET Core commands available for the terminal session in which it was run.
```

Comment:
Don't use above 
https://docs.microsoft.com/en-us/style-guide/a-z-word-list-term-collections/a/above

```suggestion
> The preceding `export` commands only make the .NET Core commands available for the terminal session in which it was run.
```

Reply:
But if you're only installing the runtime and not the SDK, why would you get .NET Core commands?

Reply:
I don't know if I've seen a proper name for them, but I think this is about commands like `dotnet --info` and `dotnet [name of executable]` that don't require the SDK.